poonam/JS/Assignment6_js/string.html

```diff
@@ -0,0 +1,152 @@
+            }
+
+            function strlength(){
+                var str1 = document.getElementById("val1").value;
+                var output1 = str1.length;
+                output1 = document.getElementById("res1").innerHTML= "Output = "+output1;
+            }
+
+            function index(){
+                var str2 = document.getElementById("val2").value;
+                var ind1 = document.getElementById("ind1").value;
+                var output2 = str2.indexOf(ind1);
+                output2 = document.getElementById("res2").innerHTML= "Output = "+output2;
+            }
+
+            function lastindex(){
+                var str3 = document.getElementById("val3").value;
+                var ind2 = document.getElementById("ind2").value;
+                var output3 = str3.lastIndexOf(ind2);
+                output3 = document.getElementById("res3").innerHTML= "Output = "+output3;
+            }
+
+            function concatstr(){
+                var cstr1 = document.getElementById("val4").value;
+                var cstr2 = document.getElementById("val5").value;
+                var output4 = cstr1.concat(cstr2);
+                output4 = document.getElementById("res4").innerHTML= "Output = "+output4;
+            }
+            function slicestr(){
+                var str5 = document.getElementById("val6").value;
+                var ind1 = document.getElementById("find").value;
+                var ind2 = document.getElementById("lind").value;
+                var output5 = str5.slice(ind1, ind2);
+                output5 = document.getElementById("res5").innerHTML= "Output = "+output5;
+            }
+            function lower_case(){
+                var str6 = document.getElementById("val7").value;
+                var output6 = str6.toLowerCase();
+                output6 = document.getElementById("res6").innerHTML= "Output = "+output6;
+            }
+            function upper_case(){
+                var str7 = document.getElementById("val8").value;
+                var output7 = str7.toUpperCase();
+                output7 = document.getElementById("res7").innerHTML= "Output = "+output7;
+            }
+            function search_str(){
+                var str8 = document.getElementById("val9").value;
+                var word = document.getElementById("word").value;
+                var output8 = str8.search(word);
+                output8 = document.getElementById("res8").innerHTML= "Output = "+output8;
+            }
+            function replace_str(){
+                var str9 = document.getElementById("val10").value;
+                var nword = document.getElementById("nword").value;
+                var rword = document.getElementById("rword").value;
+                var output9 = str9.replace(nword, rword);
+                output9 = document.getElementById("res9").innerHTML= "Output = "+output9;
+            }
+        </script>
+    </head>
+    <body>
+        <form action="">
```

Comment:
fill the action attribute please like (#)

Reply:
done